packages/datagrid/package.json

```diff
@@ -5,17 +5,14 @@
   "mainSrc": "src/index.js",
   "license": "Apache-2.0",
   "scripts": {
-    "prepare": "rimraf ./lib && babel -d lib ./src/ --ignore 'src/**/*.test.js','src/**/*.stories.js' && cpx -v \"./src/**/*.scss\" ./lib",
+    "prepare": "talend-scripts build:lib",
     "start": "start-storybook -p 6010",
-    "test": "jest --silent",
-    "test:noisy": "jest",
-    "test:watch": "jest --watch",
-    "test:cov": "jest --coverage",
+    "test": "talend-scripts test --silent",
+    "test:noisy": "talend-scripts test",
+    "test:watch": "talend-scripts test --watch",
+    "test:cov": "talend-scripts test --coverage",
     "test:demo": "build-storybook",
-    "lint:es": "eslint --config ../../.eslintrc --ignore-path ../../.eslintignore src",
-    "lint": "npm run lint:es",
-    "storybook": "start-storybook -p 6010",
-    "build-storybook": "build-storybook"
```

Comment:
Just to be sure, not used in CI?

Reply:
It's the `test:demo` script that is used. :)